packages/instrumentation-azure/package.json

```diff
@@ -39,7 +39,7 @@
     "@opentelemetry/core": "^1.22.0",
     "@opentelemetry/instrumentation": "^0.49.0",
     "@opentelemetry/semantic-conventions": "^1.22.0",
-    "@traceloop/ai-semantic-conventions": "^0.5.6"
+    "@traceloop/ai-semantic-conventions": "*"
```

Comment:
didn't you lock it to specific version in the previous pr?

Reply:
It will get locked once I publish